src/Particle/SoaDistanceTableAAOMPTarget.h

```diff
@@ -120,6 +120,13 @@ struct SoaDistanceTableAAOMPTarget : public DTD_BConds<T, D, SC>, public Distanc
   const DistRow& getOldDists() const override { return old_r_; }
   const DisplRow& getOldDispls() const override { return old_dr_; }
 
+  const RealType* getMultiWalkerTempDataPtr() const override
```

Comment:
What's the reason for this raw pointer?  There are no docs here and the documentation in the base class just says its to the "full distance table data"  But here it is to mw_mem->nw_new_old_dist_displ.data().  But I don't find it used.

I couldn't get an idea of what is going on from just reading DistanceTable code so there is likely some documentation on its mw resources needed as well.

Reply:
My bad. I copied the code but didn't update the documentation for getMultiWalkerTempDataPtr.
Currently I need two functions to access multi walker data
```
getMultiWalkerTempDataPtr
getMultiWalkerDataPtr
```
Optimally it should be replaced with proper containers. Thank you for the question. I have some ideas now but will probably try to clean them up after I finish the J2 offload work.
Documentation will be updated in the next PR.

Reply:
Ok